kubernetes_practice/resources/server.py

```diff
@@ -0,0 +1,41 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+"""
+This is a sample http server
```

Comment:
It is good practice to use ONLY english when writing code and document. But this docstring needs to be expanded a little bit. Let's sync tonight.